[en]: kant update user guide (#5104)
DaoCloud/DaoCloud-docs: docs/en/docs/kant/user-guide/device/index.md

End device can range from sensors and controllers to smart cameras or industrial computers.

> End device can be connected to edge nodes, and connected and unified management through the Modbus protocol.

Reviewer: ```suggestion
End device can be connected to edge nodes. You can connect them through the Modbus protocol and manage them centrally.
```

## Device Model

> A device model is a digital representation of an end device in the cloud. It abstracts the common characteristics of the device into a data model, using properties to define the device and the information it can provide externally.

Reviewer 2: ```suggestion
A device model is a digital representation of an end device in the cloud. It abstracts the common characteristics
of the device into a data model, using properties to define the device and the information it can provide externally.
```
